RadixWallet/Features/TransactionReviewFeature/CustomizeFees/CustomizeFees+View.swift

```diff
@@ -159,7 +156,7 @@ extension CustomizeFees {
 				}
 
 				if let insufficientBalanceMessage = viewState.insufficientBalanceMessage {
-					WarningErrorView(text: insufficientBalanceMessage, type: .error)
+					WarningErrorView(text: insufficientBalanceMessage, type: .warning)
```

Comment:
I wonder why we have `WarningErrorView` and `Hint.ViewState.Kind.warning`. Not sure if we actually want to support both views or there should be just one (and some designs are outdated).

Reply:
I was going to say that Hint is only used in the AppTextField, but that is apparently not true anymore. So yes, that should be unified.